<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addword</class>
 <widget class="QWidget" name="addword">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/static/images/bubu.ico</normaloff>:/static/images/bubu.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#widget  {
	border:1px solid black;
	border-radius:5px;
}</string>
  </property>
  <widget class="QWidget" name="titleWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>310</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#titleWidget{
background-color:rgb(22, 166, 243);
color:black;
border-radius:5px;
border:1px solid black;
}</string>
   </property>
   <widget class="QPushButton" name="minBtn">
    <property name="geometry">
     <rect>
      <x>229</x>
      <y>1</y>
      <width>39</width>
      <height>39</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>36</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover { 
 background-color: red; 
 color: rgb(255, 255, 255);
border: 1px solid black;
	font: 36pt &quot;Microsoft YaHei UI&quot;;
}
QPushButton {background-color:#E1E1E1;
color:black;
background-color: rgb(22, 166, 243); 
border-radius:5px;
border:0px solid black;
	font: 36pt &quot;Microsoft YaHei UI&quot;;}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="closeBtn">
   <property name="geometry">
    <rect>
     <x>269</x>
     <y>1</y>
     <width>39</width>
     <height>39</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover { 
 background-color: red; 
 color: rgb(255, 255, 255);
border: 1px solid black;
font-size:36px}
QPushButton {background-color:#E1E1E1;
color:black;
background-color: rgb(22, 166, 243); 
border-radius:5px;
border:0px solid black;
font-size:36px;}</string>
   </property>
   <property name="text">
    <string>×</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>单词本</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="wordText">
   <property name="geometry">
    <rect>
     <x>71</x>
     <y>110</y>
     <width>171</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>English</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="meanText">
   <property name="geometry">
    <rect>
     <x>71</x>
     <y>180</y>
     <width>171</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>中文</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addBtn">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>258</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backBtn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>41</y>
     <width>71</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
 QPushButton:hover { 
color:rgb(0, 0, 0);
border:0px solid black;
background-color:rgb(240,240,240);
}
QPushButton {
color:rgb(149, 149, 149);
border:0px solid black;
background-color:rgb(240,240,240);
}
</string>
   </property>
   <property name="text">
    <string>&lt; 返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="infoLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>340</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(101, 101, 101)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
